CONTRIBUTING.md

```diff
@@ -275,6 +275,14 @@ By giving feedback on this work (and related supporting work), you can help us m
 Finally, if nothing brings you more satisfaction than seeing every last issue labeled and all resolved issues closed, feel free to message @cart for a Bevy org role to help us keep things tidy.
 As discussed in [*How we're organized*](#how-were-organized), this role only requires good faith and a basic understanding of our development process.
 
+### How to adopt pull requests
+
+Occasionally authors of pull requests get busy or become unresponsive. This is a natural part of any open source project. To avoid blocking these efforts, these pull requests may be *adopted*, where another contributor creates a new pull request with the same content. If there is an old Pull request without the label that is without updates, comment to the organization whether it is appropriate to add the *[S-Adopt-Me](https://github.com/bevyengine/bevy/labels/S-Adopt-Me)* label, to indicate that it can be *adopted*.
```

Comment:
I think there's two things a person can:
1. As you mentioned, they can ask an org member if a given PR could use the label (this generally happens on Discord)
2. If they want to pick it up themselves, they should likely leave a comment on the PR asking the author if they plan on returning to the PR and if they can adopt it. If the author gives permission to adopt or simply doesn't respond after some time (maybe a few days?), they may then continue to adopt the PR.

You pretty much have 1 written down here, but 2 is another thing users can do to adopt a PR (and this may sometimes even skip the labeling).

> Sidenote: I know this file doesn't necessarily follow this style, but I’d recommend splitting these longer paragraphs over multiple lines (with no empty lines between). This allows markdown to still treat it as a single paragraph, but makes it easier for reviewers to leave comments on particular sentences or phrases— both for this PR and all future ones that modify these lines.

Reply:
The 2 was resolved as you asked, can you check please?

Reply:
We followed your suggestion of splitting these longer paragraphs over multiple lines, but the CI / markdownlint (pull_request) accused as errors. So we needed to maintain a paragraph.

Reply:
Looks like the reason for the error was because of some spacing issues. It's telling you to remove trailing spaces on lines:

```diff
- Here is a sentence. 
+ Here is a sentence.
And here is another.
```

It also is suggested to add blank lines between lists and code blocks:

````diff
Here is some code:
+
```
fn foo() {}
```
````